tests/ui/associated-type-bounds/inside-adt.stderr

```diff
@@ -1,131 +1,56 @@
-error: associated type bounds are not allowed within structs, enums, or unions
+error: associated type bounds are only allowed in where clauses and function signatures, not in field type
```

Comment:
I think pluralizing "field type" would be better here too. (Hopefully there's a `position.pluralize()` or something similar that does all these at once?